go/pkg/client/bytestream_test.go

```diff
@@ -0,0 +1,294 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	"google.golang.org/grpc"
+
+	bsgrpc "google.golang.org/genproto/googleapis/bytestream"
+	bspb "google.golang.org/genproto/googleapis/bytestream"
+)
+
+var (
+	logStreams    map[string]*logStream
+	logStreamData = []byte("Hello World! This is large data to send.")
```

Comment:
The value of this variable is constant throughout its lifetime. Might as well make it official by declaring as a constant.

Reply:
Right. I must have thought of it as a string constant. Not a big deal.